native/src/modules/endpoint/reducers/cityContentReducer.js

```diff
@@ -17,14 +17,16 @@ export default (
   if (action.type === 'FETCH_CATEGORY') {
     const { language, path, depth, key, city } = action.params
     const initializedState = state || createCityContent(city)
-    const oldContent = state && state.categoriesRouteMapping[key] ? state.categoriesRouteMapping[key] : {}
+    const reuseOldContent = state && state.routeMapping[key] && state.routeMapping[key].routeType === 'category'
+    const oldContent = reuseOldContent ? state?.routeMapping[key] : {}
 
     return {
       ...initializedState,
-      categoriesRouteMapping: {
-        ...initializedState.categoriesRouteMapping,
+      routeMapping: {
+        ...initializedState.routeMapping,
         [key]: {
           ...oldContent,
+          routeType: 'category',
```

Comment:
~Shouldn't this already be part of `oldContent`? Probably necessary because of flow?~